pymonntorch/NetworkBehavior/Recorder/Recorder.py

```diff
@@ -58,9 +58,11 @@ def __init__(
     def initialize(self, object):
         super().initialize(object)
 
-        variables = self.parameter("variables", [])
-        if variables == []:
-            variables = self.parameter("arg_0", [])
+        variables = self.parameter("variables", None)
+        if variables is None:
+            variables = self.parameter("arg_0", None)
+            if variables is None:
```

Comment:
seems redundant!

Reply:
It's not the best way to handle this, but it's good enough for now.
Keep in mind that the `parameter` function applies the default value type (if not `None`) on values that it returns, therefore it turns a string into a list of its letters which is not the intended value.